### PR TITLE
feat(core/inlines): support inline `code`

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by marcoscaceres** [comment]
Apologies for force push... commit message got screwed up.

File: tests/spec/core/inlines-spec.js [hunk 1]
```diff
@@ -188,6 +188,35 @@ describe("Core - Inlines", () => {
...
+    expect(simple).toBeTruthy();
+    expect(simple.textContent).toBe("null");
+
+    // multi per line
+    const multi = doc.querySelectorAll("#multi code");
+    expect(multi.length).toBe(3);
+    const [firstCode, secondCode, thirdCode] = multi;
+    expect(firstCode.textContent).toBe("123");
+    expect(secondCode.textContent).toBe("undefined");
+    expect(thirdCode.textContent.endsWith("string")).toBeTruthy();
```
Comment (saschanaz) [inline tests/spec/core/inlines-spec.js]: Wouldn't `multi[i]` be more readable here?

File: src/core/inlines.js [hunk 2]
```diff
@@ -165,6 +170,7 @@ export function run(conf) {
       "(?:\\[\\[(?:!|\\\\|\\?)?[A-Za-z0-9\\.-]+\\]\\])",
       "(?:\\[\\[\\[(?:!|\\\\|\\?)?[A-Za-z0-9\\.-]+\\]\\]\\])",
       "(?:\\[=[^=]+=\\])", // Inline [= For/link =]
+      "(?<!`)(?:`[^`]+`)", // Inline `code`
```
Comment (saschanaz) [inline src/core/inlines.js]: Should we get a way to escape `` ` ``?

File: tests/spec/core/inlines-spec.js (hunk 1 above)
Comment (marcoscaceres) [reply tests/spec/core/inlines-spec.js]: yeah, you are right. Will fix.

File: src/core/inlines.js (hunk 2 above)
Comment (marcoscaceres) [reply src/core/inlines.js]: Thought about that too... I've not run into that in practice in specs, so didn't bother. Another one is "[= link to `thing` in other spec =]". At least this gives us the 90% use case of `true` `false` `null` `void`... etc.

Comment (saschanaz) [reply src/core/inlines.js]: We can just fall back to `<code>` if we get one, so maybe okay for now.

Comment (marcoscaceres) [reply src/core/inlines.js]: yeah, exactly.

**Pull request merged**